src/targets/gpu/CMakeLists.txt

```diff
@@ -109,7 +109,7 @@ add_library(migraphx_gpu
     compiler.cpp
     device_name.cpp
     fuse_ck.cpp
-    fuse_mlir.cpp
+    mlir_offload.cpp
```

Comment:
This shouldn't be renamed in this PR. fuse_mlir is consistent with the other fuse passes. So if we rename this like this than we should rename all the other fuse passes. However, such refactoring should go into a separate PR.

Reply:
So, should I rename 1) just file names or 2) file names + the corresponding structures?

Reply:
At this moment, I just changed file names. Let me know if I should to adapt the stuct names as well

Reply:
> I should to adapt the stuct names as well

Yes, the class names should match the filenames.